L1Trigger/L1TMuonBarrel/src/L1MuBMTrackFinder.cc

```diff
@@ -211,9 +211,9 @@ void L1MuBMTrackFinder::run(const edm::Event& e, const edm::EventSetup& c) {
 
             l1t::RegionalMuonCand rmc;
 
-            if(cand->hwEta()>-117 || cand->hwEta()<117 )
+            if(cand->hwEta()>-107 || cand->hwEta()<107 )//some upper and lower values for eta
                 rmc.setHwEta(cand->hwEta());
```

Comment:
LUTs of eta give lower values than 107. 107 is better suited. (-106,106) fine and (-104,104) coarse